internal/docs/readme.go

```diff
@@ -161,8 +208,8 @@ func writeReadme(packageRoot string, content []byte) (string, error) {
 	return aReadmePath, nil
 }
 
-func readmePath(packageRoot string) string {
-	return filepath.Join(docsPath(packageRoot), ReadmeFile)
+func readmePath(packageRoot, fileName string) string {
```

Comment:
Nit: make `fileName` the first argument to be consistent with other function signatures in this package?